### DISCUSSION TITLE
can't connect github in template page

### DISCUSSION THREAD

**Discussion by ttizze** [vercel/community#2775] [Help] [answered]
### Summary

I want to use template.
i connected github  
![image](https://github.com/vercel/community/assets/29332792/6d5d33ee-5f38-4157-a8e9-a791cdd5d3ed)
but Create Git Repository block is something wrong.
![image](https://github.com/vercel/community/assets/29332792/5c6850d5-46bb-49ce-b717-792550fa32f9)
there used to be an input form here.
please help!!
thanks!

### Steps to Reproduce

https://vercel.com/templates/next.js/nextjs-boilerplate
go here. and push deploy.

**Comment by walter-grace** [comment]
I am having the same issue I tried to use for Vercel-SubScription-Stripe and same issue. Tried to disconnect my connected account and now cant connect back either

**Comment by ttizze** [comment, marked as answer]
disconnected and reconnect again. i solved this issue.thanks!
